src/main/scala/com/typesafe/sbt/packager/docker/DockerPlugin.scala

```diff
@@ -19,6 +19,7 @@ import packager.Keys._
 import linux.LinuxPlugin.autoImport.{ daemonUser, defaultLinuxInstallLocation }
 import universal.UniversalPlugin.autoImport.stage
 import SbtNativePackager.{ Universal, Linux }
+import java.io.File
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Put imports in the beginning of the package](https://www.codacy.com/app/nepomukseiler/sbt-native-packager/file/2356068302/issues/source?bid=2040925&fileBranchId=3219012#l22)